knowledge-graph/src/main/scala/io/renku/knowledgegraph/entities/QueryParamDecoders.scala

```diff
@@ -45,6 +48,37 @@ object QueryParamDecoders {
     val parameterName: String = "type"
   }
 
+  private implicit val recentlyViewedEntityTypesParamDecoder: QueryParamDecoder[RecentEntitiesFinder.EntityType] =
```

Comment:
Do you think it's a good place for the new endpoint params to sit here together with the `entities` endpoint encoders?

Reply:
Well, don't have a strong opinion here. I thought that these two could probably share a lot of things, like page/limit and later also entityTypes etc. And they are _used_ outside of that package anyways. But can move them to the sub-package?

Reply:
Yes, you are right they share a lot. I was just looking at that from the perspective of having things tidy. But, yes, maybe a subpackage that is used by both endpoints would be a good idea.

Reply:
I now moved the class one level up. I noticed that it isn't used in that subtree anywhere but solely in the `MicroserviceRoutes` class. Hope that is ok, but can be changed any time obviously.